libs/api/src/services/scan/index.ts

```diff
@@ -234,92 +234,124 @@ export const GetPrecinctSelectionConfigResponseSchema: z.ZodSchema<GetPrecinctSe
 
 /**
  * @url /config/precinct
- * @method PUT
+ * @method PATCH
  */
-export interface PutPrecinctSelectionConfigRequest {
+export interface PatchPrecinctSelectionConfigRequest {
```

Comment:
Standardizing on `PATCH` for config changes, some of my past PRs had introduced a meaningless variation.

Reply:
I'm hoping we can autogenerate a bunch of this in the future and probably ditch meaningful HTTP methods. That said, here's what MDN says about [`PATCH`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods/PATCH) vs `PUT`:

> A `PATCH `request is considered a set of instructions on how to modify a resource. Contrast this with [`PUT`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods/PUT); which is a complete representation of a resource.

So if the URL you're `PATCH`ing represents the complete representation of the resource, you should be using `PUT`. If you really wanted to use `PATCH` you'd have a `/config` endpoint and your HTTP request body would have something like `{ "precinctSelection": … }`.

Reply:
I was just trying to standardize. Based on your info, it seems like they should all be `PUT`. But I don't know how important that switch is, so I'll leave it out here unless you insist.